test/unit/helpers/orderBy.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var OrderBy = require('../../../helpers/orderBy');
+
+describe('OrderBy', function () {
+
+	var sortFields = ['afield'];
+
+	it('no order by option passed', function () {
```

Comment:
I would prefer to have a more descriptive convention what's happening. So the test `no order by option passed` doesn't really tells about the expected results.
In my opinion context convention would be much nicer:
``` 
describe('when orderBy = null', function () {
   it('should return sortField = null');
   it('should return sortMethod = null');
   it('should return error = undefined');
});
// or further
describe('when desc is in lowercase', function () {
   it('converts desc to uppercase');
});
```
Hope you prefer that convention too because the work you doing is really good 👍

Reply:
MaciejBaj. Thanks for the compliment! Its nice to work with your teams code.

I have changed the tests to be more inline with the convention mentioned above... I think. :) It makes it much more legible.

Thanks for taking the time to review all this. :)